frontend/src/views/camp/navigation/mobile/NavSidebar.vue

```diff
@@ -33,6 +33,18 @@
           icon="mdi-format-list-bulleted-triangle"
           :to="{ name: 'camps', query: { isDetail: true } }"
         />
+        <SidebarListItem
+          :title="$tc('global.navigation.help')"
+          icon="mdi-help-circle-outline"
+          :href="helpLink"
+          target="_blank"
+        />
+        <SidebarListItem
+          :title="$tc('global.navigation.news')"
+          icon="mdi-news"
+          :href="newsLink"
+          target="_blank"
+        />
```

Comment:
Do we really want these 2 links before the camp specific buttons?
now you have to scroll down on a small device to print the camp.

Reply:
I don't mind. I think it would be strange to split the global options

Reply:
The termsOfServiceLinkTemplate has the same problem, and @manuelmeister made it consistent.
I opened another issue to fix this: https://github.com/ecamp/ecamp3/issues/4777

Reply:
I would also prefer (and intuitively look for) these points to be at the very bottom of the menu. I see the point about splitting the related options, but I don't think "my camps" is very much related to help and news. Also, help and news are two links which will probably only be used very rarely (especially since help is just the FAQ right now), while "my camps" is used relatively often. That alone justifies moving the help and news links to the bottom for me.